ndbench-dyno-plugins/src/main/java/com/netflix/ndbench/plugin/dyno/LocalHttpEndpointBasedTokenMapSupplier.java

```diff
@@ -0,0 +1,11 @@
+package com.netflix.ndbench.plugin.dyno;
+
+import com.netflix.dyno.connectionpool.impl.lb.HttpEndpointBasedTokenMapSupplier;
+
+public class LocalHttpEndpointBasedTokenMapSupplier extends HttpEndpointBasedTokenMapSupplier{
+
+	public LocalHttpEndpointBasedTokenMapSupplier(int port) {
+		super("http://{hostname}:8081/REST/v1/admin/cluster_describe",port);
```

Comment:
I used 8081 to not conflict with other servers that might be running. NDBench shall be running on 8080. Since this is pointing out to a node.js script with the dynomite seeds.